docs/npm-workspaces.md

```diff
@@ -0,0 +1,72 @@
+# npm workspaces
```

Comment:
Brilliant documentation 👍🏼 
We don't really follow creation documentation inside `docs` directory I would suggest placing this inside `libs/README.md` or append to root `README.md`, though appreciate root README.MD is getting rather large.